<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="SelectDir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QTreeView" name="ViewFS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="Cont" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>269</y>
      <width>651</width>
      <height>121</height>
     </rect>
    </property>
    <widget class="QLabel" name="Name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Size">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Размер выделенного
элемента</string>
     </property>
    </widget>
    <widget class="QListWidget" name="ViewData">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="ContCountFile" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <widget class="QLabel" name="label2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Соличество файлов</string>
      </property>
     </widget>
     <widget class="QLabel" name="CountFile">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>0</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ContCountSubDir" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <widget class="QLabel" name="label3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Соличество папок</string>
      </property>
     </widget>
     <widget class="QLabel" name="CountSubDir">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>0</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
